### ISSUE TITLE
Please upgrade to mermaid-cli v9

### ISSUE THREAD

**Issue by martin-cowie** [raghur/mermaid-filter#103]
Love your work. 

If you upgrade from v8 to v9 of mermaid-cli then you get support for Mermaid git diagrams into the deal.

**Comment by gschnellOnera** [comment]
Please upgrade to mermaid-cli **v10.1**

**Comment by zadjii-msft** [comment]
I believe this was updated to v9 in #95, but I don't think this was ever shipped as a release. 

(I'd also appreciate a bump to v10.5+ 😉)

I did manage to workaround this by:
* Manually installing the mermaid package
* Going into `...npm\node_modules\mermaid\dist` and copying `mermaid.min.js`
* Pasting that into `...npm\node_modules\mermaid-filter\node_modules\@mermaid-js\mermaid-cli`.

That somehow amazingly worked for the v10.5.0 of `mermaid` I had installed, and the `8.14.0` of `mermaid-cli` that I had was replacing. Feels really janky though, and certainly prone to breaking.

**Comment by lunarcloud** [comment]
I manually upgraded by using the override feature of packages.json - no ill effects noticed.

```json
{
  "dependencies": {
    "@mermaid-js/mermaid-cli": "^10.6.1",
    "mermaid-filter": "1.4.6",
    "pagedjs-cli": "^0.4.3"
  },
  "overrides": {
    "mermaid-filter": {
      "@mermaid-js/mermaid-cli": "~10"
    }
  }
}

```

**Comment by lunarcloud** [comment]
This worked for me, no issues:

```json
{
  "dependencies": {
    "@mermaid-js/mermaid-cli": "^10.6.1",
    "mermaid-filter": "^1.4.6",
    "pagedjs-cli": "^0.4.3"
  },
  "engines": {
    "npm": ">=8.3.0"
  },
  "overrides": {
    "mermaid-filter": {
      "@mermaid-js/mermaid-cli": "^10.6.1"
    }
  }
}
```

**Comment by raghur** [comment]
done - published 1.4.7
